src/contacts/processors/impl/ListPersonActionProcessor.java

```diff
@@ -0,0 +1,41 @@
+package contacts.processors.impl;
+
+import contacts.ConsoleReader;
+import contacts.Constants;
+import contacts.model.Record;
+import contacts.processors.*;
+import contacts.service.IRecordService;
+
+import java.io.IOException;
+import java.util.List;
+
+public class ListPersonActionProcessor implements IActionProcessor {
```

Comment:
if your processor prints single record, why do you name it ListPersonActionProcessor

Reply:
No, my processor prints list of records, where you can choose one